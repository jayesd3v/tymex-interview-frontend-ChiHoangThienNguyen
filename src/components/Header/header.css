.bg-body-tertiary {
    background-color: #181719b3 !important;
}