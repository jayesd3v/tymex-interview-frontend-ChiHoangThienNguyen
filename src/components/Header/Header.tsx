import { Button, Container, Nav, Navbar, Form } from 'react-bootstrap';
import './header.css';

const Header = () => {
    return (
        <Navbar fixed='top' expand='lg' className='bg-body-tertiary'>
            <Container>
                <Navbar.Toggle aria-controls='navbarScroll' />
                <Navbar.Collapse id='navbarScroll'>
                    <Nav className='me-auto my-2 my-lg-0' navbarScroll>
                        <Nav.Link href='/'>HOME</Nav.Link>
                        <Nav.Link href='/'>ABOUT US</Nav.Link>
                        <Nav.Link href='/'>OUR TEAMS</Nav.Link>
                        <Nav.Link href='/marketplace'>MARKETPLACE</Nav.Link>
                        <Nav.Link href='/'>WHITEPAPER</Nav.Link>
                    </Nav>
                    <Form className='d-flex'>
                        <Button variant='outline-success'>Connect Wallet</Button>
                    </Form>
                </Navbar.Collapse>
            </Container>
        </Navbar>
    );
};

export default Header;
