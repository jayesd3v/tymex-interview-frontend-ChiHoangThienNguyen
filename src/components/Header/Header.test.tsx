import { render } from "@testing-library/react";
import Header from "./Header";

describe('Header tests', () => {
    test('snapshot test', () => {
        const { container } = render(<Header />);
        expect(container).toMatchSnapshot();
    });
});
