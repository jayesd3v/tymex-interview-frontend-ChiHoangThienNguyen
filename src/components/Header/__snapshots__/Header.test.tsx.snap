// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests snapshot test 1`] = `
<div>
  <nav
    class="bg-body-tertiary navbar navbar-expand-lg navbar-light fixed-top"
  >
    <div
      class="container"
    >
      <button
        aria-controls="navbarScroll"
        aria-label="Toggle navigation"
        class="navbar-toggler collapsed"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="navbar-collapse collapse"
        id="navbarScroll"
      >
        <div
          class="me-auto my-2 my-lg-0 navbar-nav navbar-nav-scroll"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            HOME
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            ABOUT US
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            OUR TEAMS
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/marketplace"
            href="/marketplace"
          >
            MARKETPLACE
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            WHITEPAPER
          </a>
        </div>
        <form
          class="d-flex"
        >
          <button
            class="btn btn-primary"
            type="button"
          >
            Connect Wallet
          </button>
        </form>
      </div>
    </div>
  </nav>
</div>
`;
